pydis_site/apps/content/resources/guides/pydis-guides/contributing.md

```diff
@@ -91,26 +79,55 @@ Our projects on Python Discord are open source and [available on Github](https:/
   </div>
 </div>
 
-If you don't understand anything or need clarification, feel free to ask any staff member with the  **@PyDis Core Developers** role in the server. We're always happy to help!
+# How do I start contributing?
+  Unsure of what contributing to open source projects involves? Have questions about how to use GitHub? Just need to know about our contribution etiquette? Completing these steps will have you ready to make your first contribution no matter your starting point.
+
+  Feel free to skip any steps you're already familiar with, but please make sure not to miss the  [Contributing Guidelines](#5-read-our-contributing-guidelines).
+
+  If you are here looking for the answer to a specific question, check out the sub-articles in the top right of the page to see a list of our guides.
+
+  **Note:** We use Git to keep track of changes to the files in our projects. Git allows you to make changes to your local code and then distribute those changes to the other people working on the project. You'll use Git in a couple steps of the contributing process. You can refer to this [**guide on using Git**](./working-with-git/).
+  {: .notification }
+
+### 1. Fork and clone the repo
+  GitHub is a website based on Git that stores project files in the cloud. We use GitHub as a central place for sending changes, reviewing others' changes, and communicating with each other. You'll need to fork the repository to create a copy under your own GitHub account. You'll make your changes to this copy, which can then later be merged into the PyDis repository.
```

Comment:
Since this doesn't assume any Git/GitHub knowledge do we want to explain forking?

Reply:
That was sort of the intention of this clause
> to create a copy under your own GitHub account

Do you think it needs more?

Reply:
I think we can reverse the sentence: "You'll need to create a copy under your own account by forking the repository with the Fork button" (or something similar). This way, the reader can understand the meaning of forking a repository before reading "forking".